src/components/AddTodoForm.jsx

```diff
@@ -10,8 +10,13 @@ const AddTodoForm = ({ onAddTodo }) => {
 
   const handleAddTodo = (e) => {
     e.preventDefault();
-    onAddTodo({ id: Date.now(), title: todoTitle });
-    setTodoTitle("");
+    const title = todoTitle.trim();
+    if (title !== "") {
```

Comment:
Another option: you can combine the code `const title = todoTitle.trim(); if (title !== "")` and simplify it to `if (todoTitle.trim())`